strings/parse_punctuation.py

```diff
@@ -0,0 +1,39 @@
+# parse_punctuation.py
+
+import codecs
+import string
+
+
+def remove_punc(path, remove_ctrl_char=False):
```

Comment:
This looks really cool!

Please add type hints on this principle function: __def remove_punc(path: str, remove_ctrl_char: bool=False)-> str:__